app/jobs/regular/export_csv_file.rb

```diff
@@ -288,12 +288,14 @@ def write_csv_file(data, header)
             csv << value
           end
         end
+        # compress CSV file
+        `gzip --best #{File.expand_path("#{UserExport.base_directory}/#{@file_name}", __FILE__)}`
```

Comment:
Do we really need to be doing `-best`? I think -5 is the best compromise between speed / size